pkg/controller/pod/pod_controller.go

```diff
@@ -132,12 +138,20 @@ var _ reconcile.Reconciler = &ReconcilePod{}
 type ReconcilePod struct {
 	client     client.Client
 	scheme     *runtime.Scheme
-	status     *statusmanager.StatusManager
+	status     statusmanager.StatusManagerInterface
```

Comment:
I'm not sure if it's good to add 'Interface' in name

Reply:
Done. The changes:
type StatusManager struct -> type Status struct
type StatusManagerK8s struct -> type StatusK8s struct
type StatusManagerOc struct -> type StatusOc struct
type StatusManagerInterface interface -> type StatusManager interface